contents/articles/2016-11_focal-musical-expression/index.md

```diff
@@ -0,0 +1,35 @@
+--- 
+title: An Eye-Tracking Musical Expression Controller
+date: Fri Nov 11 2016 14:43:37 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Focal is an experimental concept that uses an AR headset with Pupil Labs eye tracking add-ons..."
+featured_img: "../../../../media/images/blog/focal-systemb.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/focal-music.jpg"
+---
+
+Focal is an exciting experimental concept that uses an AR headset with Pupil Labs eye tracking add-ons. Focal enables musicians to have greater freedom to manage and control electronic effects while both hands are occupied. The performer can control many virtual electronic audio effects and synthesis parameters by glancing at the effects they want to activate and using a single expression pedal to adjust those effects, while maintaining posture and balance for minimal disturbance and interaction during performance. Check out Stewart Greenhill and Cathie Travers full paper [here](http://stewartgreenhill.com/documents/FocalEyeTrackingMusicalExpressionController-NIME2016.pdf).
```

Comment:
replace:
> glancing
with **gazing**

Also this sentence is a run on sentence. Split it up into:
The musician can control many virtual electronic audio effects and synthesis parameters by fixating on on effects displayed on the AR device. An expression foot pedal can be used to adjust the selected effects, enabling the musician to maintain posture and balance for minimal disturbance during performance.